spec/requests/creating_spec.rb

```diff
@@ -0,0 +1,56 @@
+RSpec.describe "Creating" do
+  it "responds with Bad Request if there are no resources at the top level" do
+    skip("Currently results in an internal server error")
```

Comment:
😬 ??

Reply:
Yepppppp. See https://github.com/ManageIQ/manageiq-api/pull/143#discussion_r146954819 for more on this